src/Api/Extensions/ContentExtension.php

```diff
@@ -69,7 +69,7 @@ private function filterUnpublishedViewlessContent(QueryBuilder $queryBuilder): v
         $queryBuilder->setParameter('status', Statuses::PUBLISHED);
 
         //todo: Fix this when https://github.com/doctrine/orm/issues/3835 closed.
-        if (! empty($this->viewlessContentTypes)) {
+        if ($this->viewlessContentTypes->isNotEmpty()) {
```

Comment:
👍 I like this a lot, manly because of this: https://www.zachstronaut.com/posts/2009/02/09/careful-with-php-empty.html